modules/importing-a-custom-workbench-image.adoc

```diff
@@ -0,0 +1,75 @@
+:_module-type: PROCEDURE
+
+[id='importing-a-custom-workbench-image_{context}']
+= Importing a custom workbench image
+
+[role='_abstract']
+ifdef::upstream[]
+You can import custom workbench images that cater to your {productname-short} project's specific requirements. From the *Notebook images* page, you can enable or disable a previously imported workbench image and create an accelerator profile as a recommended accelerator for existing notebook images.
+endif::[]
+ifndef::upstream[]
+In addition to workbench images provided and supported by {org-name} and independent software vendors (ISVs), you can import custom workbench images that cater to your project's specific requirements.
+endif::[]
+
+After you have built a custom image and placed it in an image registry that is accessible to {productname-long}, you must import it so that your {productname-short} users (data scientists) can access it when they create a project workbench.
+
+ifndef::upstream[]
+{org-name} supports you in adding custom workbench images to your deployment of {productname-short} and ensuring that they are available for selection when creating a workbench. However, {org-name} does not support the contents of your custom workbench image. That is, if your custom workbench image is available for selection during workbench creation, but does not create a usable workbench, {org-name} does not provide support to fix your custom workbench image.
```

Comment:
{org-name} supports adding... (instead of "supports you in adding...")

Suggestion: {org-name} supports adding custom workbench images to your deployment of {productname-short}, ensuring that they are available for selection when creating a workbench.